OWNERS

```diff
@@ -10,6 +10,7 @@ approvers:
 - zshi-redhat
 - pmtk
 - pacevedom
+- stlaz
```

Comment:
Shouldn't we add him also as reviewer?

Reply:
I think approvers automatically get reviewer rights, too, but it's better to be safe.